resize-observer-1/Overview.bs

```diff
@@ -318,7 +318,7 @@ interface ResizeObservation {
     : <dfn>observedBox</dfn>
     :: Which box is being observed.
     : <dfn>lastReportedSizes</dfn>
-    :: Ordered sequence of last reported sizes.
+    :: Ordered frozen array of last reported sizes.
```

Comment:
Let's just say 'Ordered array' instead since the Frozen-ness of the array is not particularly relevant.